pom.xml

```diff
@@ -647,6 +647,12 @@
                 <version>42.1.4</version>
```

Comment:
are benchmark results from the commit message are still valid? Have you noticed any perfomance boost with `BenchmarkPlanner`?

Reply:
maybe just mention in the commit message that the benchmarks are run with a future commit.  The data is still good to have.

Reply:
okay, i am squashing now and will add some more explanation to the commit message.